### PR TITLE
Release 3.2.0-BETA.2

### PR DESCRIPTION
**Pull request by kryalama**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Add Spring integration (preview).
```
Comment (trask) [inline CHANGELOG.md:4]:
```suggestion
* Add Spring Integration instrumentation (preview).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+* Add thread name to log capture. Resolves [#1699](https://github.com/microsoft/ApplicationInsights-Java/issues/1699)
+* Use azure monitor exporter instead of Open Telemetry exporter.
+* Add data truncation and sanitization to telemetry data.
```
Comment (trask) [inline CHANGELOG.md:7]: I don't think there's any end-user facing changes here, in which case no need to mention
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+* Support W3C baggage propagation.
+* Update App Services internal diagnostics comments.
```
Comment (trask) [inline CHANGELOG.md:9]: not end-user facing
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+* Disable legacy Request-Id propagation by default.
+* Handle partial content success due to Stamp specific redirects. Resolves [#1674](https://github.com/microsoft/ApplicationInsights-Java/issues/1674)
+* Log controlled warning when telemetry truncated. Resolves [#1021](https://github.com/microsoft/ApplicationInsights-Java/issues/1021)
+* Support system properties for the main "coordinates". Resolves [#1684](https://github.com/microsoft/ApplicationInsights-Java/issues/1684)
```
Comment (trask) [inline CHANGELOG.md:13]:
```suggestion
* Support system properties for the connection string, role name and role instance. Resolves [#1684](https://github.com/microsoft/ApplicationInsights-Java/issues/1684)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Add Spring integration (preview).
+* Add thread name to log capture. Resolves [#1699](https://github.com/microsoft/ApplicationInsights-Java/issues/1699)
```
Comment (heyams) [inline CHANGELOG.md:5]: can we remove Resolves everywhere in this pr.  put link before the period.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,20 @@
...
+* Use azure monitor exporter instead of Open Telemetry exporter.
+* Add data truncation and sanitization to telemetry data.
+* Support W3C baggage propagation.
+* Update App Services internal diagnostics comments.
+* Disable legacy Request-Id propagation by default.
+* Handle partial content success due to Stamp specific redirects. Resolves [#1674](https://github.com/microsoft/ApplicationInsights-Java/issues/1674)
+* Log controlled warning when telemetry truncated. Resolves [#1021](https://github.com/microsoft/ApplicationInsights-Java/issues/1021)
+* Support system properties for the main "coordinates". Resolves [#1684](https://github.com/microsoft/ApplicationInsights-Java/issues/1684)
+* Small fixes to 2.x interop and exporter code cleanup.
```
Comment (trask) [inline CHANGELOG.md:14]:
```suggestion
* Small fixes to 2.x interop.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,20 @@
...
+* Persist regular telemetries on disk inorder to retry when failed to send telemetry.
```
Comment (trask) [inline CHANGELOG.md:15]: this is not a change (well, I guess it's a change from BETA.1, but just back to normal)
```suggestion
```

File: CHANGELOG.md (hunk 4 above)
Comment (heyams) [inline CHANGELOG.md:13]: I will be more specific about this 'coordinates', list them please.

File: gradle.properties [hunk 8]
```diff
@@ -1,3 +1,3 @@
 // Project properties
-version=3.2.0-BETA.2
+version=3.2.0-BETA.3
```
Comment (trask) [inline gradle.properties:2]: this is BETA.2 release

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
```
Comment (trask) [inline CHANGELOG.md:3]: this is BETA.2
```suggestion
# Version 3.2.0-BETA.2
```

File: CHANGELOG.md (hunk 7 above)
Comment (heyams) [inline CHANGELOG.md:15]: Add disk persistence?

File: CHANGELOG.md (hunk 9 above)
Comment (heyams) [reply CHANGELOG.md:3]: yeah, we didn't release beta.2.  @kryalama  please reuse this draft: https://github.com/microsoft/ApplicationInsights-Java/releases/tag/untagged-969a656e91fa14295250

File: gradle.properties (hunk 8 above)
Comment (kryalama) [reply gradle.properties:2]: my bad got confused

**Review by trask — APPROVED** [review]

**Pull request merged**